test/tests/transfers/multiapp_reporter_transfer/dist_vector/tests

```diff
@@ -0,0 +1,49 @@
+[Tests]
+    issues = '#26953'
+    design = 'MultiAppReporterTransfer.md'
+    [vec_rep]
+        requirement = "The system shall support the ability to transfer reporter
+        data in a N subapp to 1 parent app, and 1 parent app to N subapp fashion when"
```

Comment:
split the test between 1 to N and N to 1

Reply:
I almost want to keep at least one of these tests. This will show that sending 1 to N then receiving N to 1 will result in the same order for the vectors.

Reply:
sounds good. Then add two simple inputs for the split ones please

Reply:
I created two more tests, that only run one or the other and not both at the same time.